testnets/_IBC/sagatestnet-osmosistestnet.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "sagatestnet",
```

Comment:
the chains have to be in alphabetical order, so the file name and chain_1 would be osmosistestnet, and chain_2 sagatestnet